pkg/rasactl/status.go

```diff
@@ -20,33 +20,71 @@ import (
 
 	"github.com/RasaHQ/rasactl/pkg/status"
 	"github.com/RasaHQ/rasactl/pkg/types"
+	"helm.sh/helm/v3/pkg/release"
 )
 
-// Status prints status for a given deployment.
-func (r *RasaCtl) Status() error {
-	var d = [][]string{}
+type StatusProject string
+
+const (
+	StatusStopped    StatusProject = "Stopped"
+	StatusRunning    StatusProject = "Running"
+	StatusInstalling StatusProject = "Installing"
+	StatusUpgrading  StatusProject = "Upgrading"
+)
+
+// Return project status, helm release, and err for a given helm release
```

Comment:
```suggestion
// GetReleaseStatus returns project status, helm release, and err for a given helm release.
```

A comment describing a function has to start with the function name.

Reply:
Got it. 😊